packs/shadows-at-sundown-bestiary/aliriel.json

```diff
@@ -4427,15 +4427,26 @@
                             {
                                 "definition": [
                                     "item:magical",
-                                    "item:type:weapon",
+                                    "item:material:silver",
                                     {
                                         "or": [
-                                            "item:material:silver",
-                                            "item:weapons-shedding-bright-light"
+                                            "item:type:melee",
+                                            "item:type:weapon"
                                         ]
+                                    },
+                                    {
+                                        "not": "item:category:unarmed"
                                     }
                                 ],
-                                "label": "PF2E.IWR.Custom.MagicalSilverWeaponsOrMagicalWeaponsBrightLight"
+                                "label": "PF2E.IWR.Custom.MagicalSilverWeapons"
+                            },
+                            {
+                                "definition": [
+                                    "item:magical",
+                                    "item:type:weapon",
+                                    "item:weapons-shedding-bright-light"
```

Comment:
Should we exclude unarmed here as well, given that it's not currently automated anyway?

Reply:
Yeah, probably.